.buildkite/hooks/README.md

```diff
@@ -0,0 +1,7 @@
+Hooks in this folder are expected to come from the vtools repo at the 
+time the CI pipeline runs, as these are repository-level hooks. The 
```

Comment:
All of them are repository level hooks... The reason that `pre-command` doesn't work if it's not defined in the `redpanda` repo, is that the `pre-command` script doesn't exist before running the `setup` anchor. The `pre-exit`, instead, runs always because the `setup` anchor is taking care of moving the `.buildkite/` folder to the build's root context (i.e. `SRC_DIR`). Since the `setup` anchor always sets the build context like: `redpanda/vtools`, I suspect that we shouldn't delete it from this repo and we should delete it from `vtools`. I can help checking if this theory is correct

Reply:
I see that this was fixed in https://github.com/redpanda-data/vtools/commit/ae9fd8b770e1c0804bb2527c913b66ac005c781b so never mind